selfhost/typechecker.jakt

```diff
@@ -4560,7 +4690,7 @@ struct Typechecker {
         Function(captures, params, can_throw, return_type, block, span) => .typecheck_lambda(captures, params, can_throw, return_type, block, span, scope_id, safety_mode)
         Try(expr, catch_block, catch_name, span) => .typecheck_try(expr, catch_block, catch_name, scope_id, safety_mode, span, type_hint)
         TryBlock(stmt, catch_block, error_name, error_span, span) => .typecheck_try_block(stmt, error_name, error_span, catch_block, scope_id, safety_mode, span)
-        Operator => {
+        else => {
```

Comment:
Please don't add an else here, let's just have the compiler yell at us when we add new ones